### PR TITLE
fix(Location): extend prop typing with HTML element props

### PR DESCRIPTION
**Pull request by eszthoff**

### CODE REVIEW

File: stories/LocationSelector.tsx [hunk 1]
```diff
@@ -41,6 +41,7 @@ storiesOf('Organisms|LocationSelector', module)
 
         return (
             <LocationSelector
+                className="test-class"
```
Comment (carlobernardini) [inline stories/LocationSelector.tsx:44]: Maybe add a test?

Comment (eszthoff) [reply stories/LocationSelector.tsx:44]: what sort of test do you have in mind?

File: src/components/LocationSelector/LocationSelector.tsx [hunk 2]
```diff
@@ -19,7 +19,9 @@ import styles from './LocationSelector.scss';
 
 const { block, elem } = bem('LocationSelector', styles);
 
-interface Props extends LocationSelectorDialogWithGoogleLoaderProps {
+interface Props
+    extends LocationSelectorDialogWithGoogleLoaderProps,
+        Omit<React.HTMLAttributes<HTMLDivElement>, 'id'> {
```
Comment (suleymangezsat) [inline src/components/LocationSelector/LocationSelector.tsx:24]: Do we need to omit `id` here ? It's already defined as optional string in `HTMLAttributes`

File: stories/LocationSelector.tsx (hunk 1 above)
Comment (suleymangezsat) [inline stories/LocationSelector.tsx:44]: I guess it's forgotten to delete

File: src/components/LocationSelector/LocationSelector.tsx (hunk 2 above)
Comment (eszthoff) [reply src/components/LocationSelector/LocationSelector.tsx:24]: yes, and the definitions clash, so had to Omit it.

File: stories/LocationSelector.tsx (hunk 1 above)
Comment (eszthoff) [reply stories/LocationSelector.tsx:44]: no, it is intentional. This is the error we had that inspired this PR. TS would throw when adding a className. So as a "test" I added it to storybook. I wonder if we shouldn't do it in every story, as this is not the first time we forgot to handle it. But that is for a separate discussion I think.

File: src/components/LocationSelector/LocationSelector.tsx (hunk 2 above)
Comment (suleymangezsat) [reply src/components/LocationSelector/LocationSelector.tsx:24]: Interesting 🤔 I've tried to remove omit on my local and TS didn't give any error or warning

Comment (eszthoff) [reply src/components/LocationSelector/LocationSelector.tsx:24]: 🤔  I tried it again, and now TS is happier. Good catch. Thanks. I will remove it

File: stories/LocationSelector.tsx (hunk 1 above)
Comment (carlobernardini) [reply stories/LocationSelector.tsx:44]: A test that ensures custom html attributes can be applied (without causing errors, but I guess that's already covered with linting) like you seem to have added this line for

**Pull request merged**